guides/common/linkchecker.ini

```diff
@@ -21,3 +21,4 @@ ignore=example.com
   # This isn't published downstream yet
   access.redhat.com/documentation/en-us/red_hat_satellite/6.10/html-single/managing_configurations_using_puppet_integration/index
   access.redhat.com/documentation/en-us/red_hat_satellite/6.11
```

Comment:
You can remove these lines since by now they should be published. Also, this branch shouldn't check 6.10/6.11 docs.
```suggestion
```

Also a reminder: once 6.12 is published I'd recommend to drop the line below so the linkchecker will verify the links.

Reply:
Ack, will have the 6.12 lines removed once the docs are published.